### PR TITLE
Enhancement/angular components/fhi week model

### PR DESCRIPTION
**Pull request by yleaxeman**

### CODE REVIEW

File: projects/fhi-angular-components/src/lib/fhi-time/fhi-time-constants.ts [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
   // TODO: i18n
-  static weekpickerPlaceholder = 'åååå-u[u]';
+  static weekpickerPlaceholder = 'åååå-uu';
   static weekpickerLabel = 'Velg uke';
   static weekpickerDelimiter = '-';
+
+  static weekRangeLabelFrom = 'Fra uke';
+  static weekRangeLabelTo = 'Til uke';
+
+  static maxWeek = { year: new Date().getFullYear() + 1, week: 52 };
```
Comment (proand) [inline projects/fhi-angular-components/src/lib/fhi-time/fhi-time-constants.ts:10]: Why year + one?

**Review by proand — APPROVED** [review]
May still be some issues in the code I don't see, but everything seems to work ok, so good to go 👍

File: src/MOCK_DB_DATA/library-items/time-selectors/time-selector-weekpicker.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    <a href="${CONST.FhiAngularComponentsGithubLibUrl}/fhi-weekpicker">
+      Kildekode
+    </a>
+  </li>
+  <li>
+    <a href="${CONST.ExampleComponentsGithubUrl}/form-controls/weekpicker">
+      Demokode
+    </a>
+  </li>
+  <li>
```
[the remark this answers is not in the document]
Comment (yleaxeman) [reply src/MOCK_DB_DATA/library-items/time-selectors/time-selector-weekpicker.ts:59]: I used the Multiselect and Autosuggest as template...

File: projects/fhi-angular-components/src/lib/fhi-time/fhi-time-constants.ts (hunk 1 above)
Comment (yleaxeman) [reply projects/fhi-angular-components/src/lib/fhi-time/fhi-time-constants.ts:10]: In the odd case if currently in week 53, this is the simple solution that could be a candidate for improvement and set maxWeek to 53 current year.

**Pull request merged**